src/Snowcap/Emarsys/Client.php

```diff
@@ -927,11 +1028,11 @@ protected function send($method = 'GET', $uri, array $body = array())
      *
      * @return string
      */
-    private function getAuthenticationSignature()
+    private function getAuthenticationSignature(): string
     {
         // the current time encoded as an ISO 8601 date string
-        $created = new \DateTime();
-        $iso8601 = $created->format(\DateTime::ISO8601);
+        $created = new DateTime();
+        $iso8601 = $created->format(DateTime::ATOM);
```

Comment:
I changed that following EA prompt:
[EA] The format is not compatible with ISO-8601. Use DateTime::ATOM for compatibility with ISO-8601 instead.

Reply:
Ah, maybe the variable and comment should be updated - perhaps include that reasoning too? Reading this line by itself feels like it might be a mistake in the format used.